src/benchmarks/micro/libraries/Common/AlignedMemory.cs

```diff
@@ -24,7 +24,7 @@ private unsafe AlignedMemory(void* memory, int length)
         }
 
         public static unsafe AlignedMemory Allocate(uint length, uint alignment)
-            => new AlignedMemory(NativeMemory.AlignedAlloc(length, alignment), (int)length);
+            => new AlignedMemory(NativeMemory.AlignedAlloc((UIntPtr)length, (UIntPtr)alignment), (int)length);
```

Comment:
Builds in some environments were generating warning about the argument types being wrong for `NativeMemory.AlignedAlloc`, so added the casts. Did NOT think we should change the public signature of the `Allocate` method exposed, but if we want to do that, it's a sweeping and breaking change.

Reply:
No objections here, so marking resolved